src/atsu/src/ATSU.ts

```diff
@@ -319,4 +320,36 @@ export class Atsu {
     public targetFlightState() {
         return this.flightStateObserver.FcuSettings;
     }
+
+    public vhfDatalinkStatus() {
```

Comment:
I would suggest to put these functions in the Datalink.ts.
The Datalink.ts is responsible to simulate the communication.

It is planned that it switches between SATCOM, HF and VHF.
So maybe proxies in the ATSU.ts are better (like isRemoteStationAvailable).
The logic is handled in the Datalink, but not all datalink-functions are exposed to components outside the ATSU.